chsdi/models/vector/uvek.py

```diff
@@ -486,6 +486,38 @@ class AbgeltungWasserkraftnutzung(Base, Vector):
 register('ch.bfe.abgeltung-wasserkraftnutzung', AbgeltungWasserkraftnutzung)
 
 
+class Energieberatungsstellen (Base, Vector):
+    __tablename__ = 'energieberatungsstellen'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/energieberatungsstellen.mako'
+    __bodId__ = 'ch.bfe.energieberatungsstellen'
+    __queryable_attributes__ = ['name', 'management']
```

Comment:
please make btree indexes for queryable attributes

Reply:
It's done